QModManager/OptionsManager.cs

```diff
@@ -60,7 +63,39 @@ internal static void Postfix(uGUI_OptionsPanel __instance)
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable debug logs", Config.EnableDebugLogs, new UnityAction<bool>(value => Config.EnableDebugLogs = value), null });
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable developer mode", Config.EnableDevMode, new UnityAction<bool>(value => Config.EnableDevMode = value), null });
                 }
+                #endregion Mod Config
+
+                #region Mod List
+                ModListTab = __instance.AddTab("QMods List");
+
+#if SUBNAUTICA_STABLE
+                __instance.AddHeading(ModListTab, $"QModManager Stable for Subnautica running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#elif SUBNAUTICA_EXP
+                __instance.AddHeading(ModListTab, $"QModManager Experimental for Subnautica running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#elif BELOWZERO_STABLE
+                __instance.AddHeading(ModListTab, $"QModManager Stable for Below Zero running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#elif BELOWZERO_EXP
+                __instance.AddHeading(ModListTab, $"QModManager Experimental for Below Zero running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#endif
```

Comment:
This might look neater if we make the QMM branch and the Game Name variables instead.

Reply:
Also not sure about actually having the stable/experimental is required. @Metious what do you think?

Reply:
having "experimental" for the experimental build wouldn't hurt but I don't think adding stable is necessary.
I would also change the wording to "Running QModManager {version} for {game}".

Reply:
ok, i broke it down to this
`
if SUBNAUTICA_EXP || BELOWZERO_EXP
                __instance.AddHeading(ModListTab, $"QModManager -Experimental- running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
#else
                __instance.AddHeading(ModListTab, $"QModManager running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
#endif
`
in my Commit "[educe overhead + clearup appearance](https://github.com/SubnauticaModding/QModManager/pull/274/commits/8f2842b755b822db650acc350167cf6e4d93871a)
because Purple suggested that there no need to differ the Game (SN or BZ) anymore.
Do you agree with that @Metious ? or should i go with your version "Running QModManager {version} for {game}" ?
I personally would like to see your version as it prevents any confusion what version is really running. 
(Even when the Code in the Background is the Same or nearly the Same)